config/config.go

```diff
@@ -623,7 +630,7 @@ type Config struct {
 	// Defines the ports that will be available for the API services to bind to.
 	// This is a map of protocol to PortWhiteList. This allows per protocol
 	// configurations.
```

Comment:
@mativm02 @buraksekili what do you think of adding a small text with an example of the ENV configuration here - since it's quite different? Something like https://tyk.io/docs/tyk-oss-gateway/configuration/#securitycertificatesupstream